tests/constbuffer_ut/constbuffer_ut.c

```diff
@@ -1554,4 +1584,973 @@ TEST_FUNCTION(CONSTBUFFER_CreateFromOffsetAndSizeWithCopy_unhappy_path) /*this f
     CONSTBUFFER_DecRef(origin);
 }
 
+/*Tests_SRS_CONSTBUFFER_02_041: [ If source is NULL then CONSTBUFFER_get_serialization_size shall fail and return 0. ]*/
+TEST_FUNCTION(CONSTBUFFER_get_serialization_size_with_source_NULL_fails)
```

Comment:
>CONSTBUFFER_get_serialization_size [](http://example.com/codeflow?start=14&length=34)

Please add those guiding regoins for your manager :-)

/* CONSTBUFFER_get_serialization_size */